README.md

```diff
@@ -197,24 +160,10 @@ If you have a bug to report, reproduce it in an online code editor. For example,
 
 ## Missing features
 
-See the list of features that are still missing in the component:
-
-- [ ] render [`bindings`](https://github.com/asyncapi/asyncapi/blob/master/versions/2.0.0/asyncapi.md#fixed-fields-19)
-- [ ] render [`tags`](https://github.com/asyncapi/asyncapi/blob/master/versions/2.0.0/asyncapi.md#tagsObject)
-- [ ] render [`externalDocs`](https://github.com/asyncapi/asyncapi/blob/master/versions/2.0.0/asyncapi.md#externalDocumentationObject)
-- [ ] render [`schema properties`](https://github.com/asyncapi/asyncapi/blob/master/versions/2.0.0/asyncapi.md#properties) 
-- [ ] render [`correlationID`](https://github.com/asyncapi/asyncapi/blob/master/versions/2.0.0/asyncapi.md#correlationIdObject)
+To see features we don't support, please check [issues](https://github.com/asyncapi/asyncapi-react/issues?q=is%3Aopen+is%3Aissue+label%3Aenhancement).
 
 If you want to help us develop them, feel free to contribute.
 
-## Credits
```

Comment:
I don't think it is fair to wipe out history of the project and not remember who donated the project. We do not need huge logo as nobody from Kyma participates in project development, but we still should keep information about initial author, here or in the package.json

Reply:
Ok, I will leave the info about Kyma project, but without logo.